ruby_review_week_four.rb

```diff
@@ -2,36 +2,134 @@
 
 # 1. What is a variable?
 
+=begin
+  
+A Variable is a element where you can store and access a string, integer, boolean. Variables store a single value e.g.
+var = "Max"
+  
+=end
+
 # 2. What Ruby method can you use to make a loop from an array?
 
+# array = [1,2,3,4,5]
+
+# array.each do |arr| 
+#   puts arr
+# end
+
 # 3. In Ruby, what are booleans?
 
+# Booleans are true or false values e.g. var = true, var = false
+
 # 4. How do classes relate to objects?
 
+# Classes are like the template (structure) of objects. They create the basic overview of what each object will contain.
+
 # 5. What is an instance variable?
 
+# An instance variable is a variable that is initizled through the creation of a class e.g.
+
+class Max
+  def initialize
+    @name = "Max"
+    @age = 21
+  end
+end
+
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
 
+class Person
+  def initialize(name, age)
+    @name = name
+    @age = age
+  end
+
+  def to_string
```

Comment:
make sure you follow instructions precisely, no need to code this entire method in full